packages/test/test-utils/src/testContainerRuntimeFactoryWithDefaultDataStore.ts

```diff
@@ -23,35 +23,54 @@ const getDefaultFluidObject = async (runtime: IContainerRuntime) => {
 };
 
 /**
- * ! Note: This function is purely needed for back-compat as the constructor argument structure was changed
+ * {@link ContainerRuntimeFactoryWithDefaultDataStoreConstructor} input properties.
+ *
+ * @internal
+ */
+export interface ContainerRuntimeFactoryWithDefaultDataStoreProps {
+	defaultFactory: IFluidDataStoreFactory;
+	registryEntries: NamedFluidDataStoreRegistryEntries;
+	dependencyContainer?: any;
```

Comment:
The existing version of this in packages/framework/aqueduct/src/container-runtime-factories/containerRuntimeFactoryWithDefaultDataStore.ts has documented fields, and doesn't use type `any`: this seems like its regressing in those aspects.

Also we now have multiple types with this name: it might be nice if they mention each other in their docs if they really are related, and to use a different name if they are not.

Reply:
Note that these changes extracted the existing parameters here as-is. This was an existing `any`. I did copy over the comments for the properties that match the other types (all but `dependencyContainer`), and I did go ahead and fix the typing of `dependencyContainer`, though.